roles/publishing_common/tasks/main.yml

```diff
@@ -57,6 +65,15 @@
     virtualenv_python: python2
     state: latest
 
+- name: create the publishing virtualenv - new virtualenv in python2 folder
+  become: yes
+  become_user: "{{ venvs_owner|default('www-data') }}"
+  pip:
+    name: pip
+    virtualenv: "/var/cnx/venvs/python2/archive"
```

Comment:
Should this be `/var/cnx/venvs/python2/publishing`?

Reply:
omg

Reply:
did i mention i did all this in 1.5 hours on Friday using mostly find and replace? ;) Thank goodness for code review. 🙏